src/bin/evtx_dump.rs

```diff
@@ -326,6 +329,12 @@ fn main() {
                 .takes_value(false)
                 .help("When set, output will not be indented."),
         )
+        .arg(
+            Arg::with_name("separate-attributes")
```

Comment:
Let's consider prefixing this flag, since it has no effect in XML parsing.
Maybe something like `json-separate-attributes`?

Reply:
This is now reflective with https://github.com/forensicmatt/evtx/commit/632248502582dad619b5936f3c3f34316929d2b3